lte/protos/mconfig/mconfigs.proto

```diff
@@ -71,6 +71,37 @@ message EnodebD {
         int32 tac = 8;
         int32 cell_id = 9;
         string ip_address = 10;
+        HoAlgorithmConfig ho_algorithm_config = 16;
+        message HoAlgorithmConfig {
+            // ho algorithm config
```

Comment:
Add some more detailed description here - is there a link to a specification? This comment is just repeating what the name `HoAlgorithmConfig` already tells us. Another developer coming in to use the definitions you've established here can interpret these fields in slightly different ways - please add as much clarity here as possible.